scripts/simtest/cargo-simtest

```diff
@@ -28,12 +28,12 @@ done
 #   echo "Please commit or revert your changes to Cargo.lock before running cargo simtest"
 #   exit 1
 # fi
-cd "$STARTING_DIR"
+cd "$STARTING_DIR" || exit
```

Comment:
should this return failure then instead of success? Why can this happen btw?

Reply:
Probably can't happen unless something deletes the directory while this is executing

Reply:
might change it to `exit 1` in case it does indeed happen so it won't report a success.

Reply:
done